airbyte-integrations/connectors/destination-mssql/src/main/kotlin/io/airbyte/integrations/destination/mssql/v2/MSSQLBulkLoader.kt

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.mssql.v2
+
+import io.airbyte.cdk.load.command.Dedupe
+import io.airbyte.cdk.load.command.DestinationCatalog
+import io.airbyte.cdk.load.command.DestinationStream
+import io.airbyte.cdk.load.data.csv.toCsvHeader
+import io.airbyte.cdk.load.data.withAirbyteMeta
+import io.airbyte.cdk.load.file.azureBlobStorage.AzureBlob
+import io.airbyte.cdk.load.file.azureBlobStorage.AzureBlobClient
+import io.airbyte.cdk.load.message.StreamKey
+import io.airbyte.cdk.load.pipeline.RoundRobinInputPartitioner
+import io.airbyte.cdk.load.write.StreamStateStore
+import io.airbyte.cdk.load.write.db.BulkLoader
+import io.airbyte.cdk.load.write.db.BulkLoaderFactory
+import io.airbyte.integrations.destination.mssql.v2.config.MSSQLBulkLoadConfiguration
+import io.airbyte.integrations.destination.mssql.v2.config.MSSQLConfiguration
+import io.airbyte.integrations.destination.mssql.v2.config.MSSQLIsConfiguredForBulkLoad
+import io.github.oshai.kotlinlogging.KotlinLogging
+import io.micronaut.context.annotation.Requires
+import jakarta.inject.Singleton
+
+@Singleton class MSSQLInputPartitioner : RoundRobinInputPartitioner()
```

Comment:
checking: this + MssqlBulkLoadConfig are identical to https://github.com/airbytehq/airbyte/pull/56353/?